src/main/java/com/microsoft/lst_bench/telemetry/JDBCTelemetryRegistry.java

```diff
@@ -104,6 +104,8 @@ public void flush() {
         String currentQuery = StringUtils.replaceParameters(query, values).getStatement();
         statement.execute(currentQuery);
       }
+
+      eventsStream = Collections.synchronizedList(new ArrayList<>());
```

Comment:
I'm thinking that this class is not thread safe in any case (i.e., if there are events being added while telemetry is being flushed, you can potentially lose these events). For the time being, I think we can stop using a synchronized list, add a comment in the class javadoc stating that the registry is not thread safe, and create an issue in GitHub to tackle in the future.